.github/workflows/version.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
             ;;
         esac
 
-        if [[ ! "${VERSION}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+        if [[ ! "${VERSION}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
The alternative to this would be to update all of the `printf` statements to prefix the `v` but I think simplifying that portion as much as possible is a good idea.

Reply:
Actually, we'll need to do that.

Reply:
Changed in [`272b63a` (#279)](https://github.com/chainguard-dev/bincapz/pull/279/commits/272b63ab2fbb5a50ff1aec822df023aefa32bc53).